.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,3 @@
+const baseline = require('@mui/monorepo/.markdownlint-cli2.cjs');
```

Comment:
Would it make sense for the future to prefer ESM over commonjs?

Reply:
Yes, but I didn't use the `.mjs` option because it wouldn't lot on the VSCode extension: https://github.com/DavidAnson/vscode-markdownlint/issues/248.